### DISCUSSION TITLE
Api down :(

### DISCUSSION THREAD

**Discussion by Anish-Chanda** [lukePeavey/quotable#124] [Archived]
**Describe the bug**
Error message :- `Failed host lookup: 'api.quotable.io' (OS Error: No address associated with hostname,`  
![image](https://user-images.githubusercontent.com/96709942/172769331-64d78474-82a4-43fc-bece-2eab1cb972de.png)

**To Reproduce**
1. Open an API client (Postman or Chrome)
2. Make any API request

```HTTP
GET https://api.quotable.io/tags
```

**Comment by ghost** [comment]
+1 got the same error

**Comment by dmitry-voronoff** [comment]
Same for me

**Comment by iGerman00** [comment]
Same here

**Comment by Himanshu3101** [comment]
Please fix this...

**Comment by egoTr** [comment]
I got the same error. 😥

**Comment by aloyr** [comment]
same.
dns seems to be registered on namecheap, but there is no NS record, so no names on that domain resolve.

```
dig quotable.io ns

; <<>> DiG 9.10.6 <<>> quotable.io ns
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: SERVFAIL, id: 29747
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;quotable.io.			IN	NS

;; Query time: 135 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Thu Jun 09 23:19:16 CDT 2022
;; MSG SIZE  rcvd: 40
```

**Comment by SatyamDevv** [comment]
Same here I just started using yesterday anyone know new api like this

**Comment by rudrankriyam** [comment]
@aloyr seems to work now, can you check? 

```
; <<>> DiG 9.10.6 <<>> quotable.io ns
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 32247
;; flags: qr rd ra; QUERY: 1, ANSWER: 4, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;quotable.io.			IN	NS

;; ANSWER SECTION:
quotable.io.		3600	IN	NS	ns1.dnsimple.com.
quotable.io.		3600	IN	NS	ns2.dnsimple.com.
quotable.io.		3600	IN	NS	ns3.dnsimple.com.
quotable.io.		3600	IN	NS	ns4.dnsimple.com.

;; Query time: 153 msec
;; SERVER: 192.168.1.1#53(192.168.1.1)
;; WHEN: Fri Jun 10 12:15:09 IST 2022
;; MSG SIZE  rcvd: 124
```

**Reply by aloyr** [reply]
yes. all good now.

**Comment by usmandilmeer** [comment]
Api is not working...

**Comment by lukePeavey** [comment]
Working on it...

**Comment by lukePeavey** [comment]
# FIXED

This issue has been resolved.  It may take up to 48 hours for the DNS changes to propagate word wide. I apologize for the inconvenience!

[downfor.io/quotable.io](https://downforeveryoneorjustme.com/quotable.io)

<img width="1049" alt="image" src="https://user-images.githubusercontent.com/8286271/173108358-cc68bf33-54df-45ff-b5c7-cffba45178fa.png">

[isitdownrightnow.com/quotable.io](https://www.isitdownrightnow.com/quotable.io.html)

[<img width="525" alt="image" src="https://user-images.githubusercontent.com/8286271/173109371-4ac04e53-a95f-4dfe-83bd-33ff75d35b08.png">](https://www.isitdownrightnow.com/quotable.io.html)

Thanks to @Gal1leo03 and everyone else for reporting this!

**Reply by lukePeavey** [reply]
This issue was caused by an expired DNSSEC key. We have to manually rotate DNSSEC keys every 90 days because the domain is registered with a different company than our DNS hosting provider. 

This could be avoided in the future by moving the domain registration to DNSimple. I'll look into this and see if it can be done without any downtime.
